dts/bindings/spi/gd,gd32-spi.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2021 BrainCo Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: GigaDevice GD32 SPI
+
+compatible: "gd,gd32-spi"
+
+include: [spi-controller.yaml, pinctrl-device.yaml]
+
+properties:
```

Comment:
suggestion: make `pinctrl-0` and `pinctrl-names` required.

Reply:
done.